modules/cnv-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -0,0 +1,213 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-troubleshooting-node-network.adoc
+
+[id="cnv-troubleshooting-incorrect-policy-config_{context}"]
+= Troubleshooting: Incorrect NodeNetworkConfigurationPolicy configuration
+
+Applying a NodeNetworkConfigurationPolicy to the cluster is an easy way to apply changes to the node network configuration consistently across your entire cluster.
+However, this also means it is easy to apply an incorrect configuration.
+
+This troublshooting example investigates how to identify and correct the configuration of a failed network Policy.
```

Comment:
This seems like a little too much "fluff" -- what about "You can apply changes to the node network configuration across your entire cluster by applying a NodeNetworkConfigurationPolicy. If you apply an incorrect configuration, you can use the following example to troubleshoot and correct the failed network Policy."

Reply:
I liked that fluffy vibe... but updated as suggested because Pan is right.